pkg/servicedisc/types.go

```diff
@@ -11,8 +11,8 @@ import (
 )
 
 const (
-	// ServiceTypeProxy stands for the proxy discovery.
-	ServiceTypeProxy = "proxy"
+	// ServiceTypeProxy stands for the skysocks discovery.
+	ServiceTypeProxy = "skysocks"
```

Comment:
We should rename `ServiceTypeProxy` to `ServiceTypeSkysocks`

Reply:
Done